app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -38,6 +42,21 @@ export interface IPublishRepositorySettings {
   readonly org: IAPIUser | null
 }
 
+export type RepositoryPublicationSettings =
```

Comment:
:art: these three types could live outside the component in `app/src/models/publish-settings.ts` or similar if they are used in multiple places